assets/scss/comments-and-meta.scss

```diff
@@ -56,8 +56,8 @@
             width: 15px;
```

Comment:
px sizing not rem?

Reply:
will add a future task to go through all css and examine px usage.